src/types/ortb2.d.ts

```diff
@@ -0,0 +1,41 @@
+export namespace Ortb2 {
+  type Site = {
+    page?: string;
+    ref?: string;
+    domain?: string;
+    publisher?: {
+      domain?: string;
+    };
+    keywords?: string;
+  };
+
+  type Device = {
+    w?: number;
+    h?: number;
+    dnt?: 0 | 1;
+    ua?: string;
+    language: string;
+    sua?: {
+      source?: number;
+      platform: unknown;
+      browsers: unknown[];
+      mobile: number;
+    };
+  };
+
+  type Regs = {
+    coppa?: unknown;
+    ext?: { gdpr?: unknown; us_privacy?: unknown };
+  };
+
+  type User = {
+    ext?: unknown;
+  };
+
+  type Ortb2Object = {
```

Comment:
this isnt correct, ortb object might have site, dooh, or app, but not more than one

Reply:
Sure, I'll add the props `dooh`, `app`. I couldn't find their exact definition based on the usage, so I'll just use `unknown`. If needed, these could definitely be extended in the future.

Reply:
i think #5218 kind of covers it

Reply:
#5097 actually...